pkg/rancher-desktop/main/tray.ts

```diff
@@ -186,11 +185,52 @@ export class Tray {
      *
      * This system isn't perfect -- if the renderer window is closed when connection status changes, the info is lost.
      */
-    Electron.ipcMain.on('update-network-status', (_, status: boolean) => {
-      this.handleUpdateNetworkStatus(status).catch((err:any) => {
-        console.log('Error updating network status: ', err);
-      });
+    ipcMainProxy.on('update-network-status', this.updateNetworkStatusEvent);
+  }
+
+  private k8sStateChangedEvent = (mgr: VMBackend) => {
+    this.k8sStateChanged(mgr.state);
+  };
+
+  private settingsUpdateEvent = (cfg: Settings) => {
+    this.settings = cfg;
+    this.settingsChanged();
+  };
+
+  private updateNetworkStatusEvent = (_: Electron.IpcMainEvent, status: boolean) => {
+    this.handleUpdateNetworkStatus(status).catch((err:any) => {
+      console.log('Error updating network status: ', err);
     });
+  };
+
+  /**
+   * @description Checks for an existing instance of Tray. If one does not
```

Comment:
I don't think we usually use `@description` — not against it, just found it unusual.